inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -461,15 +461,15 @@ std::shared_ptr<ngraph::Node> V10Parser::createNode(const std::vector<ngraph::Ou
     }
 
     // Try to create operation from loaded opsets
-    GenericLayerParams params_copy = params;
+    auto version_copy = params.version
```

Comment:
Why version copy? 😄 I propose:
```suggestion
    auto version = params.version
```

Reply:
Fixed.